oasislmf/model_preparation/sum_inputs.py

```diff
@@ -231,32 +177,24 @@ def write_mapping_file(sum_inputs_df, target_dir):
     return sum_mapping_fp
 
 
-
-@oasis_log
 def get_column_selection(summary_set):
     # Summary Set Not defined in settings file
     if "oed_fields" not in summary_set.keys():
         return None
 
-
     # Select Default Grouping for either
     # ['lob', 'county', 'policy', 'state', 'location', 'prog']
     elif isinstance(summary_set['oed_fields'], str):
         if summary_set['oed_fields'] in SUMMARY_GROUPING.keys():
```

Comment:
Same as above - simpler here to use

    if summary_set['oed_fields'] in SUMMARY_GROUPING: